src/  components/Navbar.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Navbar() {
+  return (
+    <nav className="navbar navbar-dark bg-primary mb-3">
+      <div className="container">
+        <a className="navbar-brand" href="/">
```

Comment:
Remember that we don't use anymore. You should use and import it from 'react-router-dom'